manifests/params.pp

```diff
@@ -131,25 +131,28 @@
   $repoforge_exclude = absent
 
   # Drush7 Settings.
-  $drush7_url = "http://pkg.tag1consulting.com/drush/drush-7/el${::operatingsystemmajrelease}/noarch"
+  $drush7_url = "http://packagecloud.io/tag1consulting/drush7/el/${::operatingsystemmajrelease}/${::architecture}"
   $drush7_enabled = '1'
   $drush7_includepkgs = absent
   $drush7_exclude = absent
-  $drush7_gpgcheck = '1'
+  $drush7_gpgcheck = '0'
+  $drush7_repo_gpgcheck = '1'
 
   # Drush8 Settings.
-  $drush8_url = "http://pkg.tag1consulting.com/drush/drush-8/el${::operatingsystemmajrelease}/noarch"
+  $drush8_url = "http://packagecloud.io/tag1consulting/drush8/el/${::operatingsystemmajrelease}/${::architecture}"
   $drush8_enabled = '1'
   $drush8_includepkgs = absent
   $drush8_exclude = absent
-  $drush8_gpgcheck = '1'
+  $drush8_gpgcheck = '0'
+  $drush8_repo_gpgcheck = '1'
 
   # Drush9 Settings.
-  $drush9_url = "http://pkg.tag1consulting.com/drush/drush-9/el${::operatingsystemmajrelease}/noarch"
+  $drush9_url = "http://packagecloud.io/tag1consulting/drush9/el/${::operatingsystemmajrelease}/${::architecture}"
```

Comment:
We should dump the Drush 9 repo completely, drush 9 isn't intended to be installed system-wide like this.

Reply:
Sure, I was just updating it to the new repo. Do we have anything that depends on it? I don't want to break things.

Reply:
No, I think I only built a single -alpha release package before it was clear that Drush 9 would need to be installed via composer.